packages/react-components/react-tabster/src/focus/createFocusOutlineStyle.ts

```diff
@@ -87,7 +102,7 @@ export const createFocusOutlineStyle = ({
       outlineColor: tokens.colorStrokeFocus2,
       outlineRadius: tokens.borderRadiusMedium,
       // FIXME: tokens.strokeWidthThick causes some weird bugs
-      outlineWidth: '2px',
+      outlineWidth: tokens.strokeWidthThick,
```

Comment:
Here's the same token. Does it work well her?

Reply:
No. I reverted it.